File: src/main/kotlin/agendaonline/model/Token_.java
package agendaonline.model;

import javax.persistence.metamodel.SingularAttribute;

public class Token_ {
    public static volatile SingularAttribute<Token, Long> user_id;
    public static volatile SingularAttribute<Token, String> client_id;
    public static volatile SingularAttribute<Token, String> user_name;
    public static volatile SingularAttribute<Token, String> scope;
}

File: src/main/kotlin/agendaonline/model/Token.kt
package agendaonline.model

import com.fasterxml.jackson.annotation.JsonIgnore
import javax.persistence.*

data class Token(
        @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
        var user_id:Long = 0L,
        var client_id:String = "",
        var user_name:String = "",
        var scope:Array<String> = arrayOf()
)